mattermost-plugin/webapp/package.json

```diff
@@ -78,18 +79,19 @@
     "ts-loader": "9.2.8",
     "typescript": "4.6.2",
     "webpack": "5.70.0",
-    "webpack-cli": "4.9.2"
+    "webpack-cli": "4.10.0",
+    "webpack-dev-server": "4.10.0"
   },
   "dependencies": {
     "core-js": "3.21.1",
     "glob-parent": "6.0.2",
     "marked": ">=4.0.12",
     "mattermost-redux": "5.33.1",
-    "react": "^17.0.2",
-    "react-dom": "17.0.2",
-    "react-intl": "^5.24.7",
-    "react-redux": "^7.2.8",
-    "react-router-dom": "5.2.0",
+    "react": "^16.13.0",
```

Comment:
As mentioned in a commit message, these needed to be downgraded temporarily to match the web app. Once Nevy's React 17 PR goes in, we can revert these. They should all be matched with fuzzy versions though so that we don't need to match the React versions exactly